code/build.sbt

```diff
@@ -2,6 +2,8 @@ organization := "com.feth"
 
 name := "play-authenticate"
 
+javacOptions ++= Seq("-Werror")
```

Comment:
There are still lots of deprecation though - but I'd rather take care of them after this PR is merged (it's already quite big, didn't want to make things even more complicated)